<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Section 4 Session Info | Exploring the utility of human genetic interactions for gene function predictions</title>
  <meta name="description" content="Function prediction by genetic interactions." />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="Section 4 Session Info | Exploring the utility of human genetic interactions for gene function predictions" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://23verse.github.io/fungi" />
  
  <meta property="og:description" content="Function prediction by genetic interactions." />
  <meta name="github-repo" content="23verse/fungi" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Section 4 Session Info | Exploring the utility of human genetic interactions for gene function predictions" />
  
  <meta name="twitter:description" content="Function prediction by genetic interactions." />
  



<meta name="date" content="2020-11-17" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="get-started.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />











<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./index.html">FUNGI</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Overview</a></li>
<li class="chapter" data-level="2" data-path="r-packages-datasets.html"><a href="r-packages-datasets.html"><i class="fa fa-check"></i><b>2</b> R packages &amp; Datasets</a></li>
<li class="chapter" data-level="3" data-path="get-started.html"><a href="get-started.html"><i class="fa fa-check"></i><b>3</b> Get started</a></li>
<li class="chapter" data-level="4" data-path="session-info.html"><a href="session-info.html"><i class="fa fa-check"></i><b>4</b> Session Info</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="_blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Exploring the utility of human genetic interactions for gene function predictions</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="session-info" class="section level1">
<h1><span class="header-section-number">Section 4</span> Session Info</h1>
<table>
<caption><span id="tab:sessionInfo">TABLE 4.1: </span>Session info on R, OS, and attached/loaded packages</caption>
<colgroup>
<col width="12%" />
<col width="6%" />
<col width="81%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">pkg</th>
<th align="left">version</th>
<th align="left">title</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">dnet</td>
<td align="left">1.1.7</td>
<td align="left">Integrative Analysis of Omics Data in Terms of Network, Evolution and Ontology</td>
</tr>
<tr class="even">
<td align="left">dplyr</td>
<td align="left">1.0.1</td>
<td align="left">A Grammar of Data Manipulation</td>
</tr>
<tr class="odd">
<td align="left">Fang</td>
<td align="left">1.0.0</td>
<td align="left">Fang-Defined Functions</td>
</tr>
<tr class="even">
<td align="left">forcats</td>
<td align="left">0.5.0</td>
<td align="left">Tools for Working with Categorical Variables (Factors)</td>
</tr>
<tr class="odd">
<td align="left">ggforce</td>
<td align="left">0.3.2</td>
<td align="left">Accelerating 'ggplot2'</td>
</tr>
<tr class="even">
<td align="left">ggplot2</td>
<td align="left">3.3.2</td>
<td align="left">Create Elegant Data Visualisations Using the Grammar of Graphics</td>
</tr>
<tr class="odd">
<td align="left">hexbin</td>
<td align="left">1.28.1</td>
<td align="left">Hexagonal Binning Routines</td>
</tr>
<tr class="even">
<td align="left">igraph</td>
<td align="left">1.2.5</td>
<td align="left">Network Analysis and Visualization</td>
</tr>
<tr class="odd">
<td align="left">magrittr</td>
<td align="left">1.5</td>
<td align="left">A Forward-Pipe Operator for R</td>
</tr>
<tr class="even">
<td align="left">patchwork</td>
<td align="left">1.0.1</td>
<td align="left">The Composer of Plots</td>
</tr>
<tr class="odd">
<td align="left">purrr</td>
<td align="left">0.3.4</td>
<td align="left">Functional Programming Tools</td>
</tr>
<tr class="even">
<td align="left">readr</td>
<td align="left">1.3.1</td>
<td align="left">Read Rectangular Text Data</td>
</tr>
<tr class="odd">
<td align="left">stringr</td>
<td align="left">1.4.0</td>
<td align="left">Simple, Consistent Wrappers for Common String Operations</td>
</tr>
<tr class="even">
<td align="left">supraHex</td>
<td align="left">1.27.3</td>
<td align="left">supraHex: a supra-hexagonal map for analysing tabular omics data</td>
</tr>
<tr class="odd">
<td align="left">tibble</td>
<td align="left">3.0.3</td>
<td align="left">Simple Data Frames</td>
</tr>
<tr class="even">
<td align="left">tidyr</td>
<td align="left">1.1.1</td>
<td align="left">Tidy Messy Data</td>
</tr>
<tr class="odd">
<td align="left">tidyverse</td>
<td align="left">1.3.0</td>
<td align="left">Easily Install and Load the 'Tidyverse'</td>
</tr>
<tr class="even">
<td align="left">XGR</td>
<td align="left">1.1.8</td>
<td align="left">Exploring Genomic Relations for Enhanced Interpretation Through Enrichment, Similarity, Network and Annotation Analysis</td>
</tr>
<tr class="odd">
<td align="left">ape</td>
<td align="left">5.4</td>
<td align="left">Analyses of Phylogenetics and Evolution</td>
</tr>
<tr class="even">
<td align="left">assertthat</td>
<td align="left">0.2.1</td>
<td align="left">Easy Pre and Post Assertions</td>
</tr>
<tr class="odd">
<td align="left">backports</td>
<td align="left">1.1.8</td>
<td align="left">Reimplementations of Functions Introduced Since R-3.0.0</td>
</tr>
<tr class="even">
<td align="left">BiocGenerics</td>
<td align="left">0.34.0</td>
<td align="left">S4 generic functions used in Bioconductor</td>
</tr>
<tr class="odd">
<td align="left">bitops</td>
<td align="left">1.0-6</td>
<td align="left">Bitwise Operations</td>
</tr>
<tr class="even">
<td align="left">blob</td>
<td align="left">1.2.1</td>
<td align="left">A Simple S3 Class for Representing Vectors of Binary Data ('BLOBS')</td>
</tr>
<tr class="odd">
<td align="left">bookdown</td>
<td align="left">0.20</td>
<td align="left">Authoring Books and Technical Documents with R Markdown</td>
</tr>
<tr class="even">
<td align="left">broom</td>
<td align="left">0.7.0</td>
<td align="left">Convert Statistical Objects into Tidy Tibbles</td>
</tr>
<tr class="odd">
<td align="left">cellranger</td>
<td align="left">1.1.0</td>
<td align="left">Translate Spreadsheet Cell Ranges to Rows and Columns</td>
</tr>
<tr class="even">
<td align="left">cli</td>
<td align="left">2.0.2</td>
<td align="left">Helpers for Developing Command Line Interfaces</td>
</tr>
<tr class="odd">
<td align="left">colorspace</td>
<td align="left">1.4-1</td>
<td align="left">A Toolbox for Manipulating and Assessing Colors and Palettes</td>
</tr>
<tr class="even">
<td align="left">compiler</td>
<td align="left">4.0.2</td>
<td align="left">The R Compiler Package</td>
</tr>
<tr class="odd">
<td align="left">crayon</td>
<td align="left">1.3.4</td>
<td align="left">Colored Terminal Output</td>
</tr>
<tr class="even">
<td align="left">crul</td>
<td align="left">1.0.0</td>
<td align="left">HTTP Client</td>
</tr>
<tr class="odd">
<td align="left">curl</td>
<td align="left">4.3</td>
<td align="left">A Modern and Flexible Web Client for R</td>
</tr>
<tr class="even">
<td align="left">DBI</td>
<td align="left">1.1.0</td>
<td align="left">R Database Interface</td>
</tr>
<tr class="odd">
<td align="left">dbplyr</td>
<td align="left">1.4.4</td>
<td align="left">A 'dplyr' Back End for Databases</td>
</tr>
<tr class="even">
<td align="left">digest</td>
<td align="left">0.6.25</td>
<td align="left">Create Compact Hash Digests of R Objects</td>
</tr>
<tr class="odd">
<td align="left">ellipsis</td>
<td align="left">0.3.1</td>
<td align="left">Tools for Working with ...</td>
</tr>
<tr class="even">
<td align="left">evaluate</td>
<td align="left">0.14</td>
<td align="left">Parsing and Evaluation Tools that Provide More Details than the Default</td>
</tr>
<tr class="odd">
<td align="left">fansi</td>
<td align="left">0.4.1</td>
<td align="left">ANSI Control Sequence Aware String Functions</td>
</tr>
<tr class="even">
<td align="left">farver</td>
<td align="left">2.0.3</td>
<td align="left">High Performance Colour Space Manipulation</td>
</tr>
<tr class="odd">
<td align="left">fs</td>
<td align="left">1.5.0</td>
<td align="left">Cross-Platform File System Operations Based on 'libuv'</td>
</tr>
<tr class="even">
<td align="left">generics</td>
<td align="left">0.0.2</td>
<td align="left">Common S3 Generics not Provided by Base R Methods Related to Model Fitting</td>
</tr>
<tr class="odd">
<td align="left">GenomeInfoDb</td>
<td align="left">1.24.2</td>
<td align="left">Utilities for manipulating chromosome names, including modifying them to follow a particular naming style</td>
</tr>
<tr class="even">
<td align="left">GenomeInfoDbData</td>
<td align="left">1.2.3</td>
<td align="left">Species and taxonomy ID look up tables used by GenomeInfoDb</td>
</tr>
<tr class="odd">
<td align="left">GenomicRanges</td>
<td align="left">1.40.0</td>
<td align="left">Representation and manipulation of genomic intervals</td>
</tr>
<tr class="even">
<td align="left">ggnetwork</td>
<td align="left">0.5.8</td>
<td align="left">Geometries to Plot Networks with 'ggplot2'</td>
</tr>
<tr class="odd">
<td align="left">ggrepel</td>
<td align="left">0.8.2</td>
<td align="left">Automatically Position Non-Overlapping Text Labels with 'ggplot2'</td>
</tr>
<tr class="even">
<td align="left">glue</td>
<td align="left">1.4.1</td>
<td align="left">Interpreted String Literals</td>
</tr>
<tr class="odd">
<td align="left">graph</td>
<td align="left">1.66.0</td>
<td align="left">graph: A package to handle graph data structures</td>
</tr>
<tr class="even">
<td align="left">grid</td>
<td align="left">4.0.2</td>
<td align="left">The Grid Graphics Package</td>
</tr>
<tr class="odd">
<td align="left">gtable</td>
<td align="left">0.3.0</td>
<td align="left">Arrange 'Grobs' in Tables</td>
</tr>
<tr class="even">
<td align="left">haven</td>
<td align="left">2.3.1</td>
<td align="left">Import and Export 'SPSS', 'Stata' and 'SAS' Files</td>
</tr>
<tr class="odd">
<td align="left">hms</td>
<td align="left">0.5.3</td>
<td align="left">Pretty Time of Day</td>
</tr>
<tr class="even">
<td align="left">htmltools</td>
<td align="left">0.5.0</td>
<td align="left">Tools for HTML</td>
</tr>
<tr class="odd">
<td align="left">httpcode</td>
<td align="left">0.3.0</td>
<td align="left">'HTTP' Status Code Helper</td>
</tr>
<tr class="even">
<td align="left">httr</td>
<td align="left">1.4.2</td>
<td align="left">Tools for Working with URLs and HTTP</td>
</tr>
<tr class="odd">
<td align="left">IRanges</td>
<td align="left">2.22.2</td>
<td align="left">Foundation of integer range manipulation in Bioconductor</td>
</tr>
<tr class="even">
<td align="left">jsonlite</td>
<td align="left">1.7.0</td>
<td align="left">A Robust, High Performance JSON Parser and Generator for R</td>
</tr>
<tr class="odd">
<td align="left">knitr</td>
<td align="left">1.29</td>
<td align="left">A General-Purpose Package for Dynamic Report Generation in R</td>
</tr>
<tr class="even">
<td align="left">labeling</td>
<td align="left">0.3</td>
<td align="left">Axis Labeling</td>
</tr>
<tr class="odd">
<td align="left">lattice</td>
<td align="left">0.20-41</td>
<td align="left">Trellis Graphics for R</td>
</tr>
<tr class="even">
<td align="left">lifecycle</td>
<td align="left">0.2.0</td>
<td align="left">Manage the Life Cycle of your Package Functions</td>
</tr>
<tr class="odd">
<td align="left">lubridate</td>
<td align="left">1.7.9</td>
<td align="left">Make Dealing with Dates a Little Easier</td>
</tr>
<tr class="even">
<td align="left">MASS</td>
<td align="left">7.3-51.6</td>
<td align="left">Support Functions and Datasets for Venables and Ripley's MASS</td>
</tr>
<tr class="odd">
<td align="left">Matrix</td>
<td align="left">1.2-18</td>
<td align="left">Sparse and Dense Matrix Classes and Methods</td>
</tr>
<tr class="even">
<td align="left">memoise</td>
<td align="left">1.1.0</td>
<td align="left">Memoisation of Functions</td>
</tr>
<tr class="odd">
<td align="left">misc3d</td>
<td align="left">0.8-4</td>
<td align="left">Miscellaneous 3D Plots</td>
</tr>
<tr class="even">
<td align="left">modelr</td>
<td align="left">0.1.8</td>
<td align="left">Modelling Functions that Work with the Pipe</td>
</tr>
<tr class="odd">
<td align="left">munsell</td>
<td align="left">0.5.0</td>
<td align="left">Utilities for Using Munsell Colours</td>
</tr>
<tr class="even">
<td align="left">nlme</td>
<td align="left">3.1-148</td>
<td align="left">Linear and Nonlinear Mixed Effects Models</td>
</tr>
<tr class="odd">
<td align="left">osfr</td>
<td align="left">0.2.8</td>
<td align="left">Interface to the 'Open Science Framework' ('OSF')</td>
</tr>
<tr class="even">
<td align="left">parallel</td>
<td align="left">4.0.2</td>
<td align="left">Support for Parallel computation in R</td>
</tr>
<tr class="odd">
<td align="left">pillar</td>
<td align="left">1.4.6</td>
<td align="left">Coloured Formatting for Columns</td>
</tr>
<tr class="even">
<td align="left">pkgconfig</td>
<td align="left">2.0.3</td>
<td align="left">Private Configuration for 'R' Packages</td>
</tr>
<tr class="odd">
<td align="left">plot3D</td>
<td align="left">1.3</td>
<td align="left">Plotting Multi-Dimensional Data</td>
</tr>
<tr class="even">
<td align="left">polyclip</td>
<td align="left">1.10-0</td>
<td align="left">Polygon Clipping</td>
</tr>
<tr class="odd">
<td align="left">R6</td>
<td align="left">2.4.1</td>
<td align="left">Encapsulated Classes with Reference Semantics</td>
</tr>
<tr class="even">
<td align="left">RCircos</td>
<td align="left">1.2.1</td>
<td align="left">Circos 2D Track Plot</td>
</tr>
<tr class="odd">
<td align="left">Rcpp</td>
<td align="left">1.0.5</td>
<td align="left">Seamless R and C++ Integration</td>
</tr>
<tr class="even">
<td align="left">RCurl</td>
<td align="left">1.98-1.2</td>
<td align="left">General Network (HTTP/FTP/...) Client Interface for R</td>
</tr>
<tr class="odd">
<td align="left">readxl</td>
<td align="left">1.3.1</td>
<td align="left">Read Excel Files</td>
</tr>
<tr class="even">
<td align="left">reprex</td>
<td align="left">0.3.0</td>
<td align="left">Prepare Reproducible Example Code via the Clipboard</td>
</tr>
<tr class="odd">
<td align="left">Rgraphviz</td>
<td align="left">2.32.0</td>
<td align="left">Provides plotting capabilities for R graph objects</td>
</tr>
<tr class="even">
<td align="left">rlang</td>
<td align="left">0.4.7</td>
<td align="left">Functions for Base Types and Core R and 'Tidyverse' Features</td>
</tr>
<tr class="odd">
<td align="left">rmarkdown</td>
<td align="left">2.3</td>
<td align="left">Dynamic Documents for R</td>
</tr>
<tr class="even">
<td align="left">rstudioapi</td>
<td align="left">0.11</td>
<td align="left">Safely Access the RStudio API</td>
</tr>
<tr class="odd">
<td align="left">rvest</td>
<td align="left">0.3.6</td>
<td align="left">Easily Harvest (Scrape) Web Pages</td>
</tr>
<tr class="even">
<td align="left">S4Vectors</td>
<td align="left">0.26.1</td>
<td align="left">Foundation of vector-like and list-like containers in Bioconductor</td>
</tr>
<tr class="odd">
<td align="left">scales</td>
<td align="left">1.1.1</td>
<td align="left">Scale Functions for Visualization</td>
</tr>
<tr class="even">
<td align="left">stats4</td>
<td align="left">4.0.2</td>
<td align="left">Statistical Functions using S4 Classes</td>
</tr>
<tr class="odd">
<td align="left">stringi</td>
<td align="left">1.4.6</td>
<td align="left">Character String Processing Facilities</td>
</tr>
<tr class="even">
<td align="left">tidyselect</td>
<td align="left">1.1.0</td>
<td align="left">Select from a Set of Strings</td>
</tr>
<tr class="odd">
<td align="left">tools</td>
<td align="left">4.0.2</td>
<td align="left">Tools for Package Development</td>
</tr>
<tr class="even">
<td align="left">tweenr</td>
<td align="left">1.0.1</td>
<td align="left">Interpolate Data for Smooth Animations</td>
</tr>
<tr class="odd">
<td align="left">utf8</td>
<td align="left">1.1.4</td>
<td align="left">Unicode Text Processing</td>
</tr>
<tr class="even">
<td align="left">vctrs</td>
<td align="left">0.3.2</td>
<td align="left">Vector Helpers</td>
</tr>
<tr class="odd">
<td align="left">withr</td>
<td align="left">2.2.0</td>
<td align="left">Run Code 'With' Temporarily Modified Global State</td>
</tr>
<tr class="even">
<td align="left">xfun</td>
<td align="left">0.16</td>
<td align="left">Miscellaneous Functions by 'Yihui Xie'</td>
</tr>
<tr class="odd">
<td align="left">XML</td>
<td align="left">3.99-0.5</td>
<td align="left">Tools for Parsing and Generating XML Within R and S-Plus</td>
</tr>
<tr class="even">
<td align="left">xml2</td>
<td align="left">1.3.2</td>
<td align="left">Parse XML</td>
</tr>
<tr class="odd">
<td align="left">XVector</td>
<td align="left">0.28.0</td>
<td align="left">Foundation of external vector representation and manipulation in Bioconductor</td>
</tr>
<tr class="even">
<td align="left">yaml</td>
<td align="left">2.2.1</td>
<td align="left">Methods to Convert R Data to YAML and Back</td>
</tr>
<tr class="odd">
<td align="left">zlibbioc</td>
<td align="left">1.34.0</td>
<td align="left">An R packaged zlib-1.2.5</td>
</tr>
</tbody>
</table>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="get-started.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "github", "twitter", "linkedin", "weibo", "instapaper", "vk"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"search": false,
"info": false
});
});
</script>

</body>

</html>
